pages/integrations/sso/adfs.md.erb

```diff
@@ -6,15 +6,15 @@ Active Directory Federation Services (ADFS) can be used as an SSO provider for y
   <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
-### Step 1. Create a Buildkite SSO Provider
+## Step 1. Create a Buildkite SSO Provider
 
 Click the Buildkite Organization Settings' Single Sign On menu item, then choose the ADFS provider from the available options:
 
-<%= image "sso-settings.png", width: 1458/2, height: 926/2, alt: "Screenshot of the Buildkite SSO Settings Page" %>
+<%= image "sso-settings.png", width: 872, height: 456, alt: "Screenshot of the Buildkite SSO Settings Page" %>
```

Comment:
Still trying to pin down exact screenshot stuff, but we probably need the larger screenshot / 2 here and elsewhere.